dev/devicelab/manifest.yaml

```diff
@@ -146,6 +146,20 @@ tasks:
     stage: devicelab
     required_agent_capabilities: ["mac/android"]
 
+  tiles_scroll_perf_ios__timeline_summary:
+    description: >
+      Measures the runtime performance of the tiles tab in the Complex Layout sample app on Android.
```

Comment:
This is not on Android, also applies to below comment.

Reply:
Done.